app/src/main/res/layout/tutorial_slide1.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
okay so the tutorial_slide 1 to 4 are very similar, except the fact that the views point to different resources.
I suggest we just use a single .xml file and customise it for different slides. This would reduce the number of lines of code significantly.